ynab_sdk/utils/clients/cached_client.py

```diff
@@ -11,23 +12,45 @@ class CachedClient(BaseClient):
 
     def __init__(self, config: CachedConfig):
         super().__init__(config)
-        self.redis = Redis(host=config.redis_host, port=config.redis_port, db=config.redis_db)
+        self.redis = Redis(host=config.redis_host, port=config.redis_port, db=config.redis_db, password=config.redis_pass)
+        self._redis_ttl: Union[int, None] = 3600
```

Comment:
Optional[int] its the correct typing here (Has the same meaning, but a matter of semantics)

Reply:
Done